cmake/public/LoadHIP.cmake

```diff
@@ -94,6 +94,27 @@ ELSE()
   SET(MIOPEN_PATH $ENV{MIOPEN_PATH})
 ENDIF()
 
+# ROCPRIM_PATH
```

Comment:
Wondering if this section should come from code like below:

# rocPRIM
--
2 |  
3 | The rocPRIM is a header-only library providing HIP parallel primitives for developing
… |  
74 | target_link_libraries(<your_target> roc::rocprim_hip)

Reply:
not clear what you mean? we link the rocprim libraries cmake returns.